resources/snippets.json

```diff
@@ -401,6 +401,18 @@
     "code": ".main-trackList-disabled{background:#f004}.main-trackList-disabled:focus-within,.main-trackList-disabled:hover{background:#f005}.main-trackList-disabled.main-trackList-selected,.main-trackList-disabled.main-trackList-selected:hover{background:#f006}",
     "preview": "resources/assets/snippets/More-Visible-Unplayable-Tracks"
   },
+  {
+    "title": "[Global Nav Bar] remove scrolling overflow",
+    "description": "Removes the max visible width restrictions of the 'sidebar' items. Useful if you have more than the scrolling limit but not as much that it will overflow with the searchbar",
+    "code": ".custom-navlinks-scrollable_container{max-width: unset!important}",
+    "preview": "resources/assets/snippets/glob-nav-max-items.png"
+  },
+  {
+    "title": "[Global Nav Bar] marketplace header margin fix",
```

Comment:
huh. this margin exists so the buttons are clickable. if you unset it, they won't be clickable again

Reply:
I have had certain themes make the menu overlap. though I have not disabled this in a long time so I'm not sure if that is still the case.